RELEASE_NOTES.md

```diff
@@ -1,5 +1,7 @@
 ## New
+- Display payment method icon in front of favorites.
 
-This is the initial release of Adyen Test Cards for Android 🥳
-
-See the [README](./README.md) for more information on how to install and use the app.
+## Fixed
+- Spaces are no longer included when autofilling a number. Thanks to @dhartwich1991!
```

Comment:
I think the tag won't work in the md file, when i try to preview it, it just looks like normal text

Reply:
I think it will work once we publish the release